superset-frontend/src/components/TableLoader/index.tsx

```diff
@@ -17,25 +17,14 @@
  * under the License.
  */
 import React, { useState, useEffect, useMemo } from 'react';
-import PropTypes from 'prop-types';
 import { t, SupersetClient, JsonObject } from '@superset-ui/core';
 import TableView from 'src/components/TableView';
-import withToasts from '../messageToasts/enhancers/withToasts';
-import Loading from './Loading';
-import '../../stylesheets/reactable-pagination.less';
-import { EmptyWrapperType } from './TableView/TableView';
+import withToasts from 'src/messageToasts/enhancers/withToasts';
+import Loading from '../Loading';
```

Comment:
```suggestion
import Loading from 'src/components/Loading';
```
If we move components around in the future, these consistent src paths should make things less painful.